File: exercice2/main.php
<?php
class Furniture {

  public $editor;
  public $designer;
  public $creation;

  function __construct($editor, $designer, $creation) {
    $this->editor = $editor;
    $this->designer = $designer;
    $this->creation = $creation;
  }
}

$wassily = new Furniture("Knoll", "Marcel Breuer", 1926);

// echo($wassily->editor. " " . $wassily->designer . " " . $wassily->creation);
print_r($wassily);
?>

File: exercice2/main.java
class Playground {
    public static void main(String[ ] args) {
        Furniture wassily = new Furniture("Knoll", "Marcel Breuer", 1926);
        System.out.println(wassily.editor + " " + wassily.designer + " " + wassily.creation);
    }
}

class Furniture {

  public  String editor;
  public String designer;
  public int creation;

  public Furniture(String editor, String designer, int creation) {
      
    this.editor = editor;
    this.designer = designer;
    this.creation = creation;
  }
}
